<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePlayer</class>
 <widget class="QMainWindow" name="SimplePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QPushButton" name="openUrl">
      <property name="text">
       <string>Open URL</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;7&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="VlcWidgetSeek" name="seek" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="VlcWidgetVideo" name="video" native="true"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="VlcWidgetVolumeSlider" name="volume" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="stop">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenLocal"/>
    <addaction name="actionOpenUrl"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMedia"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionOpenLocal">
   <property name="text">
    <string>Open local file</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVolumeSlider.h</header>
   <container>1</container>
   <slots>
    <slot>mute()</slot>
    <slot>volumeUp()</slot>
    <slot>volumeDown()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcWidgetSeek</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetSeek.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SimplePlayer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
